Lib/test/test_shutil.py

```diff
@@ -1696,6 +1696,22 @@ def _boo(filename, extract_dir, extra):
         self.assertEqual(get_unpack_formats(), formats)
 
 
+class TestStrPathArchives(_ArchiveTests, unittest.TestCase):
+
+    def _make_archive_with_path(self, *args, **kwargs):
+        return make_archive(*args, **kwargs)
+
+    _make_archive = _make_archive_with_path
+
+
+class TestPathlibPathArchives(_ArchiveTests, unittest.TestCase):
+
+    def _make_archive_with_path(self, path, /, *args, **kwargs):
+        return make_archive(pathlib.Path(path), *args, **kwargs)
+
+    _make_archive = _make_archive_with_path
```

Comment:
No need to add a class. A single test will suffice.

Reply:
Reverted; I agree that after the conversion moved into `make_archive` frontend, there is no need to explicitly test `_make_*` backends.